build-scripts/rcm-guest/push-to-mirrors.sh

```diff
@@ -10,7 +10,21 @@ set -e
 # VARIABLES
 ############
 SYMLINK_NAME="${1}"
-FULL_VERSION="${2}"
+
+# There are two VERSION_ARG formats we accept: major.minor.patch-release and major.minor-release.
+# If it is the latter, we need to add .0 as the patch for full version.
+VERSION_ARG="${2}"
+# Split VERSION_ARG between version-release
+VERSION=$(echo "${VERSION_ARG}" | cut -d '-' -f 1)
+RELEASE=$(echo "${VERSION_ARG}" | cut -d '-' -f 2-)
+
+# If 4.4 is passed in, make it 4.4.0
+if [[ "$VERSION" =~ ^[0-9]\.[0-9]$ ]]; then
```

Comment:
I think the version comes in with `v` prefix e.g. http://download.eng.bos.redhat.com/rcm-guest/puddles/RHAOS/AtomicOpenShift/4.4/v4.4.0-201911011549_2019-11-01.4/

Reply:
The `v` is added in the script, oddly enough. Invocation parameters: `ssh ocp-build@rcm-guest.app.eng.bos.redhat.com sh -s latest 4.4-202002261016 pre-release` and where it is added https://github.com/openshift/aos-cd-jobs/blob/7c9281a0ff5b05b772ffd355b451d86d7e2367ae/build-scripts/rcm-guest/push-to-mirrors.sh#L91